test_fixtures/general_repo/.idea/readme.txt

```diff
@@ -0,0 +1 @@
+Test folder for tests/test_general_validators/test_has_no_directions_from_blackkist.py
```

Comment:
`.idea` and other IDE-specific files shouldn't be in the repo: we can't rely that someone specifically uses PyCharm to see the readme.

Also, no need to indicate what the folder is for, only its contents matter.